app/components/samples/table_component.html.erb

```diff
@@ -1,17 +1,34 @@
 <% if @has_samples && @pagy.vars[:size].positive? %>
+  <% if @show_metadata_fields_size_warning %>
+    <div class="mb-4" role="status" aria-live="polite">
+      <%= viral_alert do %>
+        <span class="inline">
+          <%= @metadata_fields_size_warning_message %>
+
+          <% if @abilities[:edit_sample_metadata] %>
+            <%= link_to t("components.samples.table_component.create_template_link"),
+                        namespace_project_metadata_templates_path,
```

Comment:
Suggestion (blocking): Shouldn't this be using your method added in`app/components/samples/table_component/url_helpers.rb`